docs/schema.md

```diff
@@ -764,6 +785,7 @@ Including a URL and an email address in `credits[].contact[]`:
 			"https://twitter.com/JaninaKowalska01",
 			"mailto:nina@kowalska-family.net"
 		],
+    "type": "REMEDIATION DEVELOPER",
```

Comment:
nit once again: please fix the indent :)

Reply:
VSCode refuses to remember to switch to tabs 😭 fixed